server/models/ConfigModel/index.js

```diff
@@ -0,0 +1,22 @@
+import defaultUserConfig from '../../configs/defaultUserConfig'
+import { collections } from '../../utils/mongoCollections'
+
+const ConfigModel = {
+  withDefaults: (overrides = {}) => ({
+    config: defaultUserConfig,
+    name: 'default',
+    type: 'matrix',
+    created: new Date().toUTCString(),
+    ...overrides,
+  }),
+  save: async (db, configAttributes) => {
+    const config = ConfigModel.withDefaults(configAttributes)
+
+    return await db.collection(collections.configs).insertOne(config)
+  },
+  findConfig: async (db, configQuery) => {
```

Comment:
similar to my comment on the user model, I think this `find` method could just take the `uid` and construct the query.

```js
const config = await ConfigModel.find(db, uid)
```

```js
const ConfigModel = {
  find: async (db, uid) => await db.collection(collections.configs).findOne({ owner: uid }),
}
```

Reply:
[25454a1](https://github.com/AMP-SCZ/dpdash/pull/293/commits/25454a1384e02684707ecd6ca148e3dc2ad4032f)